manifests/quick-start/base/webhooks/github.com-secret.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: github.com
+  annotations:
+    kubernetes.io/service-account.name: github.com
+type: kubernetes.io/service-account-token
```

Comment:
Why does this secret exist in the vended manifests? SeviceAccount Secrets are generated automatically